qdatetime.cc

```diff
@@ -0,0 +1,20 @@
+#include "qdatetime.h"
+
+int qdatetime_get_datetime(lua_State *L) {
+  const QDateTime time(QDateTime::currentDateTime());
+  const char* const format_raw = luaL_checkstring(L, 1);
+  const QString format(format_raw);
+  const QString time_string(time.toString(format));
+  const char* const time_string_raw = qPrintable(time_string);
+  lua_pushstring(L, time_string_raw);
+  return 1;
```

Comment:
@LevchenkoS this function need to be int? I can't see cases where you return anything else except 1